docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,14 @@
+# 6. Chaining Operators
+
+## TLDR:
+
+- `pipe` is a functional programming concept.
+- Use `Observable` [`pipe`](API).
+- You can create many small chains to shorten larger chains.
+
+## Chaining Operators
+
+If an operator, as state above, is a "thing that can perform am operation", on a collection, then we might be able to go back to our [collection of apples](#Collections). Let's had a machine that would sort the apples (bad and good), and we also had a machine that could peel apples, and perhaps a machine that could slice the apples. We could then, in theory, chain them all together with some sort of conveyor and sort, peel, and slice the apples all in one step.
```

Comment:
Typo: "as state above" should be "as stated above".

Reply:
Type: "perform am operation" should be "perform an operation"

Reply:
Typo: Something's missing here:  "Let's had a machine". Maybe "Let's say we had a machine"?